sync/awsdatabricks.py

```diff
@@ -163,19 +161,36 @@ def create_prediction_for_run(
 
 
 def get_cluster_report(
-    cluster_id: str, plan_type: str, compute_type: str
+    run_id: str, plan_type: str, compute_type: str
```

Comment:
@gseyffert-sync, we talked about the CLI taking a run ID instead of a cluster ID because it's not so easy to get the cluster ID. I think this is the right way to do it such that the CLI continues to sit on the library's developer (public) interface.

I'd love to get your take - this is all contained in one commit so it's easy to roll back.

Reply:
yeah, I think this looks great :+1: thanks for doing this!